apps/trading/e2e/tests/wallet/test_wallet.py

```diff
@@ -0,0 +1,116 @@
+import pytest
+import re
+import json
+from playwright.sync_api import Page, expect, Route
+from vega_sim.service import VegaService
+from conftest import init_vega
+from fixtures.market import setup_continuous_market
+
+order_size = "order-size"
+order_price = "order-price"
+place_order = "place-order"
+order_side_sell = "order-side-SIDE_SELL"
+market_order = "order-type-Market"
+tif = "order-tif"
+expire = "expire"
+
+
+@pytest.fixture(scope="module")
+def vega(request):
+    with init_vega(request) as vega:
+        yield vega
+
+
+@pytest.fixture(scope="module")
+def continuous_market(vega):
+    return setup_continuous_market(vega)
+
+def handle_route_connection_lost(route: Route, request):
+        if request.method == "POST" and re.match(r"http://localhost:\d+/api/v2/requests", request.url):
```

Comment:
could the regex pattern be stored in a variable, it is referenced three times here

Reply:
yeah good idea